app/lib/multistep/form.rb

```diff
@@ -66,13 +66,13 @@ def completed_steps=(value)
     end
 
     def step_completed?(step)
-      completed_steps.include?(step)
+      completed_steps&.include?(step)
```

Comment:
Would you be able to provide some context as to why `completed` can now be `nil`?